mark-me-up-1.html

```diff
@@ -1,11 +1,192 @@
+            pariatur,
+            ipsa deleniti odio blanditiis minima deserunt aliquid! Deleniti eum necessitatibus maxime reiciendis quos
+            corporis tenetur impedit beatae illum ipsum, sapiente libero assumenda dolores praesentium temporibus,
+            nesciunt
+            et debitis est tempora quia. Cum at exercitationem asperiores facilis vel atque sint, consectetur deleniti
+            nihil
+            aut dolor! Exercitationem labore, voluptatibus natus sint ea quisquam distinctio iste modi quia ipsum
+            quaerat
+            repudiandae necessitatibus aliquam.
+
+            <table border="3" cellpadding="8" cellspacing="0">
+                <caption> <strong> grocery table</strong></caption>
+
+                <thead>
+                    <tr>
+                        <th>Sr. no.</th>
+                        <th>Name</th>
+                        <th>Item</th>
+                        <th>Rs</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                        <td rowspan="3">1.</td>
+                        <td rowspan="3">Ramesh</td>
+                        <td>sugar</td>
+                        <td>70</td>
+                    </tr>
+
+                    <tr>
+                        <td>ghee</td>
+                        <td>56</td>
+                    </tr>
+
+                    <tr>
+                        <td>oil</td>
+                        <td>40</td>
+                    </tr>
+
+                    <tr>
+                        <td rowspan="4">2.</td>
+                        <td rowspan="4">Suresh</td>
+                        <td>Spices</td>
+                        <td>70</td>
+                    </tr>
+
+                    <tr>
+                        <td>milk</td>
+                        <td>60</td>
+                    </tr>
+
+                    <tr>
+                        <td>bread</td>
+                        <td>20</td>
+                    </tr>
+
+                    <tr>
+                        <td>eggs</td>
+                        <td>49</td>
+                    </tr>
+
+                    <tr>
+                        <td rowspan="2">3.</td>
+                        <td rowspan="2">Danny</td>
+                        <td>suger</td>
+                        <td>70</td>
+                    </tr>
+
+                    <tr>
+                        <td>Wheat</td>
+                        <td>33</td>
+                    </tr>
+
+                    <tr>
+                        <th colspan="3">
+                            Total
+                        </th>
+                        <td>565</td>
+                    </tr>
+
+                </tbody>
+            </table>
+
+        </article>
+    </section>
+
+    <br>
+
+    <footer>
+        <button style="cursor: pointer;" onclick="location.href='#'">to top</button>
```

Comment:
wrong tag used. ANCHOR is appropriate here instead of a button. ANCHOR - for navigation, BUTTON - for executing script